### PR TITLE
Workshop

### PR DESCRIPTION
**Pull request by JasonWu0506**
issue IDL https://github.com/JasonWu0506/pingcap-newbie-workshop/issues/1
fix doc

### CODE REVIEW

File: doc/template-design [hunk 1]
```diff
@@ -3,4 +3,9 @@ The name of the function is CmpThenAdd. The function will compare the value of t
 
 If the value of the first parameter is greater than the value of the second parameter, the function will return the value of the first parameter.
 If the value of the second parameter are greater than the value of the first parameter, the function will return the value of the second parameter.
-Otherwise, the function will return the value of the first parameter plus the second parameter.
\ No newline at end of file
+Otherwise, the function will return the value of the first parameter plus the second parameter.
+
```
Comment (jebter) [inline doc/template-design:7]: No associated issue

File: doc/template-design [hunk 2]
```diff
@@ -3,4 +3,9 @@ The name of the function is CmpThenAdd. The function will compare the value of t
...
+
+
+
+https://pingcap.feishu.cn/docx/A5qddCc2QoC8jvxxLnqcJLDlnqf
```
Comment (baiyuqing) [inline doc/template-design]: This doc refer to private link.

**Review by jiezhu2007 — COMMENTED** [review]
no access of the doc

**Review by jebter — APPROVED** [review]
LGTM

**Pull request merged**